services/venice-controller/src/main/java/com/linkedin/venice/controller/stats/SystemStoreCheckStats.java

```diff
@@ -0,0 +1,21 @@
+package com.linkedin.venice.controller.stats;
+
+import com.linkedin.venice.stats.AbstractVeniceStats;
+import io.tehuti.metrics.MetricsRepository;
+import io.tehuti.metrics.Sensor;
+import io.tehuti.metrics.stats.Count;
+
+
+public class SystemStoreCheckStats extends AbstractVeniceStats {
+  private final Sensor badSystemStoreCount;
+
+  public SystemStoreCheckStats(MetricsRepository metricsRepository, String name) {
+    super(metricsRepository, name);
+
+    badSystemStoreCount = registerSensorIfAbsent("bad_system_store_count", new Count());
```

Comment:
I expect only a few system stores are going to be bad. If we are going to do that it will be two queues though. And we do have logs printing out the bad store names.

Reply:
There is no need to maintain two queues.
We just need to scan the unhealthy store sets and break them down into two categories before reporting metrics.